### PR TITLE
Fixed windows network path (\\server\file.exe) and random err

### PR DESCRIPTION
**Pull request by xarkes**
* Set pid to -1 when the reason is EXIT_PID
* Removed extra newline
* More verbose error message
* Added support to network files (debugger did not work when reopening a file which was remote)

### CODE REVIEW

File: libr/util/file.c [hunk 1]
```diff
@@ -174,6 +174,10 @@ R_API char *r_file_abspath(const char *file) {
 		if (cwd && *file != '/')
 			ret = r_str_newf ("%s"R_SYS_DIR"%s", cwd, file);
 #elif __WINDOWS__ && !__CYGWIN__
+		// Network path
+		if (!strncmp(file, "\\\\", 2)) {
```
Comment (radare) [inline libr/util/file.c]: Missing space

File: libr/util/str.c [hunk 2]
```diff
@@ -1789,6 +1789,9 @@ R_API char **r_str_argv(const char *cmdline, int *_argc) {
 				case '"':
 				case ' ':
 				case '\\':
+#if __WINDOWS__ && !__CYGWIN__
```
Comment (radare) [inline libr/util/str.c]: Wtf?

Comment (xarkes) [inline libr/util/str.c]: idk

**Comment by radare** [comment]
I think you want to handle \\ here, not \c and others, right?

**Comment by xarkes** [comment]
Yes but it's ok because it's after a backslash, so \c would go in the default case

**Pull request merged**